bin/pipeline_result_processor/prp/models/typing.py

```diff
@@ -39,3 +68,16 @@ class TypingResultCgMlst(ResultMlstBase):
 
     n_novel: int = Field(0, alias="nNovel")
     n_missing: int = Field(0, alias="nNovel")
+
+
+class TypingResultLineage(ResultLineageBase):
+    """Lineage results"""
+
+    main_lin: str
+    sublin: str
```

Comment:
Nitpicking, but perhaps better to keep consistent here, so `sub_lin` rather than `sublin`?

Reply:
No worries, sublineage is one word. Hence not needing to add an underscore:)